dags/public-health/covid19/sync_covid_testing_data.py

```diff
@@ -0,0 +1,99 @@
+"""
+Pull data from MOPS COVID Dashboard and upload to ESRI
+"""
+import datetime
+import os
+
+import arcgis
+import pandas as pd
+
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.S3_hook import S3Hook
+from airflow.operators.python_operator import PythonOperator
+
+
+def get_data(filename, workbook, sheet_name):
+    df = pd.read_excel(workbook, sheet_name=sheet_name, skiprows=1, index_col=0)
+    df = df.T
+    df = df.iloc[:, 1:3]
+    df.reset_index(level=0, inplace=True)
+    df.columns = ["Date", "Performed", "Test Kit Inventory"]
+    df["Date"] = pd.to_datetime(df["Date"], errors="coerce")
```

Comment:
We have found that AGOL really works better when the timestamp is UTC (after a lot of effort and tears working with the other COVID DAGs). Would it make sense to you to do something like
```python
df["Date"]  = pd.to_datetime(df["Date"]).dt.tz_localize("US/Pacific").dt.normalize().dt.tz_convert("UTC")
```
?

Reply:
So I specifically set the time zone to PST when creating the original layer in AGOL. It seemed to update correctly when I ran the script to update the layer. I think we should probably test as is, and if we need to correct then we can add this

Reply:
Okay, let's see. I have found the timezone interpretation in AGOL to be somewhat inscrutable and under-documented